app/synchronizationerror.cpp

```diff
@@ -37,6 +37,16 @@ SynchronizationError::ErrorType SynchronizationError::errorType( int errorCode,
     // Project no longer exists / is on different server
     return ErrorType::ProjectNotFound;
   }
+  else if ( errorCode == 422 )
+  {
+    // Hit the maximun number of contributor per month
+    if ( errorMessage.contains( QStringLiteral( "Maximum number of workspace contributors" ) ) )
```

Comment:
This option is possible, but it does not give us the flexibility to change the description on server later on (as clients would expect this exact string). Instead, we need to check for the error code `MonthlyContributorsLimitHit` ([see server code](https://github.com/MerginMaps/server-private/blob/fda7fc34bdbad91827391ebf4f525f4abd0c8e5b/server/src/service/errors.py#L70)).

You probably do not have that code available in this place, so we have two options, either
 1. pass the error code to this method from `merginapi`, or
 2. (less preferable) emit the failure signal directly from `merginapi`, similarly to how we do it for the storage limit, see https://github.com/MerginMaps/mobile/blob/404ac2b379289be2d1c5b387e1e2a11bc69dcdb2/core/merginapi.cpp#L2135